src/llama-vocab.cpp

```diff
@@ -1530,6 +1570,32 @@ std::vector<llama_vocab::id> llama_tokenize_internal(const llama_vocab & vocab,
     return output;
 }
 
+llm_tokenizer * llama_get_tokenizer(const llama_vocab & vocab) {
```

Comment:
`create` would be a more appropriate name:

```suggestion
llm_tokenizer * llama_create_tokenizer(const llama_vocab & vocab) {
```